Refresh.GameServer/Types/OAuth/OAuth2ErrorResponse.cs

```diff
@@ -0,0 +1,13 @@
+using JetBrains.Annotations;
+
+namespace Refresh.GameServer.Types.OAuth;
+
+#nullable disable
```

Comment:
We shouldn't have this be nullable disabled, since we actually have nullable properties. Using a `= null!;` for `Error` is probably okay.